### ISSUE TITLE
ActionView::Template::Error (uninitialized constant Arel::Nodes::Top) after upgrade to 3.5.3

### ISSUE THREAD

**Issue by corbolt** [mastodon/mastodon#19804]
### Steps to reproduce the problem

After Updating to 3.5.3 and restarting the services, im no longer able to view the about-page of my server, due to an "ActionView::Template::Error (uninitialized constant Arel::Nodes::Top)" error.

### Expected behaviour

the page displays as usual

### Actual behaviour

its not working (500)

### Specifications

Mastodon 3.5.3

**Comment by ClearlyClaire** [comment]
Can you provide the full stack trace?

**Comment by unitof** [comment]
I am on 3.5.3 and getting same error on /about page, on an instance with currently just one user (myself).

**Comment by dweekly** [comment]
Also seeing this issue after upgrading to 3.5.3.

**Comment by ClearlyClaire** [comment]
Thank you for the trace! I think you just need to do `RAILS_ENV=production ./bin/tootctl cache clear`. If you updated from before v3.5.0, that is required, as per the release notes.

**Comment by dweekly** [comment]
Huzzah! Confirmed that that fixes it. Thank you, @ClearlyClaire.

Just for context I started out with the "official" DigitalOcean droplet, which uses a VERY old version of Mastodon. I just did a git pull and checkout of 3.5.3; it can be a little challenging for new Mastodon server operators to read through all the docs and know how to cleanly upgrade things.

Are there ways the software could detect when a new version of Mastodon is running and old caches are present, such as by making the cache directory paths tied to the Mastodon build version? That way others won't have to run into this kind of problem in the future.
